wovn_index_sample.php

```diff
@@ -20,5 +20,12 @@
 # Set 404 status code if file not included
 if (!$included) {
   header($_SERVER["SERVER_PROTOCOL"] . " 404 Not Found");
-  echo "Page Not Found";
+
+  # Look for 404.html file in the root directory
+  $paths_404 = [dirname(__FILE__) . "/404.html", ];
+  $included_404 = wovn_helper_include_by_paths_with_ssi($paths_404);
+
+  if (!$included_404) {
+    echo "Page Not Found";
+  }
```

Comment:
@hythof @a-bougouin Is this a reasonable thing to do?

Reply:
(Please view the file diff--the 4 line preview does not show all of the logic that I'm concerned about.)

Reply:
with_ssi or without_ssi?

Reply:
By default please use the non-ssi inclusion.

> Is this a reasonable thing to do?

I am ok with this.